src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/hibernate/configuration/configuration_checklist): Add a private default constructor to HibernateUtil class in order to prevent creating HibernateUtil objects.